app/controllers/organisation_relationships_controller.rb

```diff
@@ -18,12 +18,36 @@ def housing_providers
     end
   end
 
+  def add_housing_provider
+    organisations = Organisation.all
+    respond_to do |format|
+      format.html do
+        @organisations = organisations
+        render "organisation_relationships/add_housing_provider", layout: "application"
+      end
+    end
+  end
+
+  def create
+    @resource = OrganisationRelationship.new(child_organisation_id: @organisation.id, parent_organisation_id: related_organisation_params, relationship_type: 0)
+    @resource.save!
+    redirect_to housing_providers_organisation_path
+  end
+
 private
 
   def organisation
     @organisation ||= Organisation.find(params[:id])
```

Comment:
Remember that only a support user should be able to create a relationship for any org.

Reply:
I thought data coordinators could too?

Reply:
They can create a relationship _to_ any org, but not _from_ any org.